qa/integration/specs/spec_helper.rb

```diff
@@ -15,6 +15,16 @@
 # specific language governing permissions and limitations
 # under the License.
 
+def es_allow_wildcard_deletes(es_client)
+  es_client.cluster.put_settings body: { transient: { 'action.destructive_requires_name' => false } }
+end
+
+def clean_es(es_client)
+  es_client.indices.delete_template(:name => "*")
+  es_client.indices.delete(:index => "*")
```

Comment:
I don't know if this is too wide or could be parameterized, but I don't foresee test cases where we want to drop only a subset of the indices.

Reply:
If it does, we can always add to it later.